glyphs2fontir/src/source.rs

```diff
@@ -1633,6 +1633,37 @@ struct ColorGlyphsWork {
     font_info: Arc<FontInfo>,
 }
 
+impl ColorGlyphsWork {
+    fn color_glyph(&self, glyph_name: &SmolStr) -> &glyphs_reader::Glyph {
+        self.font_info
+            .font
+            .glyphs
+            .get(glyph_name)
+            .expect("Only valid color glyphs names")
+    }
+}
+
+/// Creates a [PaintGlyph] for a split glyph, that is one that is known to have 1..N shapes with the same paint
+fn create_paint_glyph(
+    default_master_id: &str,
+    color_glyph: &glyphs_reader::Glyph,
+) -> Result<PaintGlyph, Error> {
+    let Some(default_layer) = color_glyph
+        .layers
+        .iter()
+        .find(|l| l.layer_id == default_master_id)
+    else {
+        return Err(Error::BadGlyph(BadGlyph::new(
+            color_glyph.name.clone(),
+            BadGlyphKind::MissingMaster(default_master_id.to_string()),
+        )));
+    };
+    Ok(PaintGlyph {
+        name: color_glyph.name.clone().into(),
+        paint: to_ir_paint(color_glyph.name.clone(), &default_layer.shapes[0])?,
+    })
```

Comment:
I presume it's just temporary that you are getting the PaintGlyph shape from the default layer, until proper variable support?